configfiles/test-spd-911.xml

```diff
@@ -46,6 +46,9 @@
       </Connections911Params>
     </ConnectionsParams>
 
+    <InputParams class="InputManager" name="InputParams">
```

Comment:
Is there a factory class for this? If not, then the class isn't needed.

Reply:
Removed.